sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/query.sql

```diff
@@ -20,6 +20,7 @@ WITH combined AS (
     END
     AS provider,
     match_type,
+    mozfun.norm.os(metadata.user_agent.os) AS normalized_os,
```

Comment:
That's good to know. Meaning that I don't need to normalize `metadata.user_agent.os` again here?

Also, looks like `metadata.user_agent.os` comes with the OS version for certain OSes, such as Windows. Would `SPLIT(metadata.user_agent.os, ' ')[OFFSET(0)] AS normalized_os` be sufficient here?

Reply:
> Would SPLIT(metadata.user_agent.os, ' ')[OFFSET(0)] AS normalized_os be sufficient here

I think so. Although you should check what it does for null handling. I expect it will return null on null input, but there could be an error thrown by SPLIT or by the OFFSET function. Also see `SAFE_OFFSET`.